src/tests/ftest/util/cont_security_test_base.py

```diff
@@ -25,37 +25,18 @@
 import os
 import pwd
 import grp
-import random
+import re
 from apricot import TestWithServers
+from avocado import fail_on
 
 import dmg_utils
 from daos_utils import DaosCommand
+from command_utils import CommandFailure
+import security_test_base as secTestBase
+#TODO: Uncomment 'general_utils' once the PR1484 has landed.
+#import general_utils
 
-PERMISSIONS = ["", "r", "w", "rw", "rwc", "rwcd", "rwcdt", "rwcdtT"]
-
-
-def acl_entry(usergroup, name, permission):
-    """Create a daos acl entry for the specified user or group and permission.
-
-    Args:
-        usergroup (str): user or group.
-        name (str): user or group name to be created.
-        permission (str): permission to be created.
-
-    Return:
-        str: daos pool acl entry.
-
-    """
-    if permission == "random":
-        permission = random.choice(PERMISSIONS)
-    if permission == "nonexist":
-        return ""
-    if "group" in usergroup:
-        entry = "A:G:" + name + "@:" + permission
-    else:
-        entry = "A::" + name + "@:" + permission
-    return entry
-
+PERMISSIONS = ["r", "w", "rw", "rwc", "rwcd", "rwcdt", "rwcdtT"]
```

Comment:
'o' (set-owner), 'a' (get-acl) and 'A' (set-acl) are also valid container permissions.
'c' (create container) is a valid pool permission but not a valid container permission.

Reply:
Done, thank you for noticing this.